examples/react-navigation/android-fragment/android/app/src/main/java/com/example/bottomnavigationviews/MainActivity.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.bottomnavigationviews
```

Comment:
Again, the package name is still the default from the project template. It would be better as something like `com.example.bugsnag.reactnative.fragments`